halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/PluginProfileFactory.java

```diff
@@ -30,11 +31,20 @@
 @Slf4j
 @Component
 public class PluginProfileFactory extends StringBackedProfileFactory {
-  @Override
-  protected void setProfile(
-      Profile profile,
-      DeploymentConfiguration deploymentConfiguration,
-      SpinnakerRuntimeSettings endpoints) {
+
+  public Profile getProfileByName(
```

Comment:
this change is related to updating the manifest.yml to handle deploying resources to all services. if we don't update manifest.yml to support all services, we don't have to do this! 👍